framework/console/controllers/AssetController.php

```diff
@@ -326,6 +327,11 @@ protected function loadTargets($targets, $bundles)
     protected function buildTarget($target, $type, $bundles)
     {
         $tempFile = $target->basePath . '/' . strtr($target->$type, ['{hash}' => 'temp']);
+        if (!is_file($tempFile)) {
+            FileHelper::createDirectory(dirname($tempFile), 0755, true);
```

Comment:
Like following:
```php
if (!file_exists($target->basePath)) {
    FileHelper::createDirectory($target->basePath, 0755, true);
}
```
However placing hardcoded access mask here (0755) is not a good practice. `AssetManager::$dirMoode` should be used here.